### ISSUE TITLE
Line ignored

### ISSUE THREAD

**Issue by patxoca** [reidrac/z80count#11]
Running `z80count' on:

```asm
PLY_InterruptionOn call PLY_Init
	ld hl,PLY_Interruption_Convert
PLY_ReplayFrequency ld de,0
	ld a,d
	ld (PLY_Interruption_Cpt + 1),a
```

does not add annotations to lines 1 and 3. I suspect a problem with the regex used to identify lines.

I'll take a look later, when I finish the work on improving comment alignment.

**Comment by reidrac** [comment]
The regex may not be perfect for all possible syntax; keep in mind I used with mostly with SDCC assembler that is not the most conventional (eg. indexing with ix and iy is a bit awful).

**Comment by patxoca** [comment]
You're right, the labels in the examples are not valid in SDCC, missing colon. Apart from that, there is a bug in the way `Parser.lookup` handles labels. The PR fixes the bug.

**Comment by reidrac** [comment]
Fixed by #14
